### ISSUE TITLE
Errors generated for file and line history if a branch name contains a `/` character

### ISSUE THREAD

**Issue by jherico** [DonJayamanne/gitHistoryVSCode#333]
I tend to categorize my Git branches with paths like `fix/bug1720`, `fix/android/ndk18` and `feature/vulkan`.  When I try to do a line history for a file in Visual Studio Code while on such a branch, I see output like

```
git branch  (completed in 0.036s)
git shortlog -e -s -n HEAD  (completed in 0.612s)
git log --full-history --pretty=oneline --date-order --decorate=full --skip=0 --max-count=100 fix/android/ndk18 --follow -- libraries/gpu/src/gpu/Config.slh  (completed in 1.48s)
git show-ref  (completed in 0.048s)
git status -sb  (completed in 0.054s)
git remote get-url origin/fix/android  (completed in 0.032s)
fatal: No such remote 'origin/fix/android'
```

It looks like the code generating the line `git remote get-url origin/fix/android ` is incorrectly deciding the remote name is everything before the final slash in the `remoteName/branchName` string.  

I'd suggest that you switch to everything before the first slash, but this would break for people who use forward slashes in their remote names, although I would suspect this would be a smaller number of people.  The only real solution I can think of would be to examine the remote names and branch names and hope there's no ambiguity, i.e  remote `foo` branch `bar/baz` and remote `foo/bar` with branch `baz`.

**Comment by ole1986** [comment]
I assume this should be fixed with PR #330

**Comment by DonJayamanne** [comment]
Seems to be fixed.
